dspace-api/src/test/java/org/dspace/builder/ClarinLicenseResourceMappingBuilder.java

```diff
@@ -37,6 +38,21 @@ private ClarinLicenseResourceMappingBuilder create(final Context context) {
         return this;
     }
 
+    public static void delete(Integer id) throws Exception {
+        if (Objects.isNull(id)) {
+            return;
+        }
+        try (Context c = new Context()) {
+            c.turnOffAuthorisationSystem();
```

Comment:
?

Reply:
what has this with provenance?

Reply:
That is a *Builder class used in integration tests. This is the common way the delete method looks in Vanilla.

Reply:
but why is it in this PR?

Reply:
OK, in our case, the c.turnOffAuthorisationSystem(); line would not be present because the functions below can be called without authorization. I have removed that line.